local-tests/prometheus/prometheus.yml

```diff
@@ -6,3 +6,14 @@ scrape_configs:
   - job_name: 'prometheus'
     static_configs:
       - targets: ['localhost:9090']
+
+  - job_name: 'amp'
+    metrics_path: '/amp/metrics'
+    params:
+      sap-client: [001]
+      scenario: [example-prom]
+    basic_auth:
+      username: 'DEVELOPER'
```

Comment:
Could you change the credentials to env variables like in the telegraf config and add a note in the readme?

Reply:
@JohannesKonings it looks like prometheus does not support env vars für prometheus.yml
https://stackoverflow.com/questions/50009932/environment-variables-in-prometheus-values

I removed the credentials with placeholders

Reply:
OK, we should try to document this somwhere